Fix: If two images are posted within the same second, only the first one will be saved.

KhushamBansal opened the pull request:
<!-- Thanks for submitting a PR! Please ensure the following requirements are met in order for us to review your PR -->

## Checklist

- [x] Ensure your code works with manual testing.
- [x] New or updated `public`/`exported` symbols have accurate [TSDoc](https://tsdoc.org/) documentation.
- [x] Linter and other CI checks pass.
- [x] I have licensed the changes to Element by completing the [Contributor License Agreement (CLA)](https://cla-assistant.io/element-hq/element-desktop)

### My approach to solve the issue:
Added a timestamp to the filename:
Used Date.now() to get the current time in milliseconds.
Modified the targetFileName to include the timestamp, ensuring uniqueness.
Closes element-hq/element-web#29079

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/element-hq/element-desktop?pullRequest=2075) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/element-hq/element-desktop?pullRequest=2075) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/element-hq/element-desktop?pullRequest=2075) it.</sub>

MidhunSureshR reviewed (changes requested):
Not sure what this is fixing, please provide some more context

KhushamBansal commented:
**Here is the issue I have work on:** https://github.com/element-hq/element-desktop/issues/1812
**My approach to solve the issue:**
Added a timestamp to the filename:
Used Date.now() to get the current time in milliseconds.
Modified the targetFileName to include the timestamp, ensuring uniqueness.

dbkr commented:
Hello! You've updated the PR title but the description still links to the bug that this PR doesn't fix. I don't think there is a bug for the behaviour this PR is actually changing, and I'm not really sure how much of a problem it would be: most browsers will automatically append a '(1)' or similar if a file already exists by the same name. This would mean all downloaded files get a timestamp appended which doesn't seem desirable.

I think the best thing to do here might to apply a similar change to fix the original bug, which would be in the element-web repo, which I think would be https://github.com/element-hq/element-web/blob/a0044d6b5f841acd48870dcc99ff6f27f0248f61/src/utils/exportUtils/HtmlExport.tsx#L378 and corresponding places in the other exporters, ideally.

dbkr commented:
Going to go & close this now as it's been a week. Suggestion would be to open a PR on element-web anyway, as per above.
